traffic_ops/experimental/go-api/tenant/EndPointSeeder.go

```diff
@@ -0,0 +1,202 @@
+package tenant
+
+/*
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+ 
+import (
+    "database/sql"
+    "encoding/json"
+    "net/http"
+    "fmt"
+    "time"
+
+    "github.com/gorilla/mux"
+  _ "github.com/lib/pq"
+)
+
+
+//TODO(nirs) polymorphisem
+
+/////BASE - COMMON TO ALL SEEDERS (needs to move to another package)
+/// Defintions
+type EndpointSeeder struct{
+    myDb        *sql.DB
+}
+
+/// Interface
+func NewEndpointSeeder(aDb *sql.DB) *EndpointSeeder {
+    endPointSeeder := new(EndpointSeeder)
+    endPointSeeder.myDb = aDb
+    return endPointSeeder
+}
+
+func (aEndpointSeeder *EndpointSeeder) Seed (aRouter *mux.Router, aApiPrefix string) {
+
+    aRouter.HandleFunc(aApiPrefix+"/tenants", aEndpointSeeder.getListEndpoint).Methods("GET")
```

Comment:
Consider changing the method strings "GET", "POST", etc to use the constants in https://golang.org/pkg/net/http/#pkg-constants so string typos are impossible.